odoo/addons/base/data/res_currency_rate_demo.xml

```diff
@@ -727,10 +727,10 @@
             <field name="rate">1105.24376765</field>
         </record>
 
-        <record forcecreate="0" id="rateZWL" model="res.currency.rate">
-            <field name="currency_id" ref="ZWL" />
+        <record forcecreate="0" id="rateZIG" model="res.currency.rate">
+            <field name="currency_id" ref="ZIG" />
             <field name="name">2010-01-01</field>
```

Comment:
```suggestion
            <field name="name">2024-04-05</field>
```

Reply:
Yeah, it shouldn't be in 2010 as the currency is new :) but it should be the 8th of April according to Wikipedia.
<img width="1033" alt="image" src="https://github.com/odoo/odoo/assets/50015226/c46914da-80c6-468a-a544-5d7cd7bc4863">